heudiconv/dicoms.py

```diff
@@ -392,20 +393,15 @@ def embed_nifti(dcmfiles, niftifile, infofile, bids_info, min_meta):
         # may be odict now - iter to be safe
         stack = next(iter(stack))
 
-        #Create the nifti image using the data array
-        if not op.exists(niftifile):
-            nifti_image = stack.to_nifti(embed_meta=True)
-            nifti_image.to_filename(niftifile)
-            return ds.NiftiWrapper(nifti_image).meta_ext.to_json()
-
         orig_nii = nb.load(niftifile)
         aff = orig_nii.affine
         ornt = nb.orientations.io_orientation(aff)
         axcodes = nb.orientations.ornt2axcodes(ornt)
         new_nii = stack.to_nifti(voxel_order=''.join(axcodes), embed_meta=True)
-        meta = ds.NiftiWrapper(new_nii).meta_ext.to_json()
-
-    meta_info = None if min_meta else json.loads(meta)
+        meta_file = ds.NiftiWrapper(new_nii).meta_ext.to_json()
+        meta_info = json.loads(meta_file)
+    else:
+        meta_info = {}
 
     if bids_info:
```

Comment:
We should also clean up this check:
- L408 - 412 should be removed
- L414 as well